app/views/notes/_card.html.erb

```diff
@@ -53,9 +53,9 @@
         <li><a>Total Likes: <%= node.likers.length %></a></li>
         <div class="content" style="width: 100%" >
           <% if @compact.nil? %>
-           
+
             <li class="buttons mb-2"><% if @current_user && ['admin','moderator'].include?(@current_user.role) %><a style="width: 90px" class="btn btn-outline-secondary" href="/moderate/spam/<%= node.id %>"><i class="fa fa-ban"></i> <%= t('notes._notes.spam') %></a><% end %></li>
-            <li class="buttons"><a style="width: 90px;" class="btn btn-outline-secondary" href="<%= node.path %>"><%= t('notes._notes.read_more') %></a></li>
+            <li class="buttons"><a style="width: 100px;" class="btn btn-outline-secondary" href="<%= node.path %>"><%= t('notes._notes.read_more') %></a></li>
```

Comment:
Setting a fixed width is the problem here.
![Screenshot_20200209_102411](https://user-images.githubusercontent.com/35997844/74099685-73b05d80-4b26-11ea-9792-dcae6bcc4463.png)

The text is not centred. The inline style should be removed.